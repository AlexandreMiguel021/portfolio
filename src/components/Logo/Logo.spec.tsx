import { render } from '@testing-library/react'
import { Logo } from '.'
import '@testing-library/jest-dom'

describe('Logo', () => {
	it('should render a Logo correctly', () => {
		const { container } = render(<Logo />)
		expect(container).toMatchSnapshot()
	})

	it('should render a logo with alexandre title', () => {
		const { getByText } = render(<Logo />)
		expect(getByText('Alexandre')).toBeInTheDocument()
	})
})
