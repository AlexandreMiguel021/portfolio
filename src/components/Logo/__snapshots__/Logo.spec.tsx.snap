// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render a Logo correctly 1`] = `
<div>
  <a
    class="logo"
    data-aos="flip-up"
    href="/"
  >
    <span>
      Alexandre
    </span>
  </a>
</div>
`;
